pymc4/_random_variables.py

```diff
@@ -1,12 +1,75 @@
+"""
+Implements the RandomVariable base class (and the necessary BackendArithmetic).
+Wraps selected tfp.distributions (listed in __all__) as pm.RandomVariables.
+Implements random variables not supported by tfp as distributions.
+"""
+
 from . import _template_contexts as contexts
 
-import tensorflow as tf
+import sys
+import numpy as np
 import tensorflow_probability as tfp
-
-__all__ = ["Normal", "HalfNormal", "Multinomial", "Dirichlet"]
+from tensorflow_probability import distributions as tfd
+
+
+# Random variables that PyMC4 support, but tfp does not support as
```

Comment:
You mean PyMC3? PyMC4 currently doesn't have any of these.

Reply:
Oops sorry, typo!